examples/creatives/base_simid_creative.js

```diff
@@ -138,6 +139,16 @@ class BaseSimidCreative {
     this.simidProtocol.resolve(eventData, {});
   }
 
+  /**
+   * Called when the creative receives a resize message from the player.
+   * @param {!Object} eventData Data from the event.
+   * @protected
+   */
+  onReceiveResize(eventData) {
+    this.environmentData.creativeDimensions = eventData.args.creativeDimensions;
+    this.environmentData.videoDimensions = eventData.args.videoDimensions;
+  }
```

Comment:
would you want to resolve here?

Reply:
In this case you wouldn't need to because resize is information only.
https://interactiveadvertisingbureau.github.io/SIMID/#reference-table